### ISSUE TITLE
🐛 in `Reverse`

### ISSUE THREAD

**Issue by babur001** [useAnimations/react-useanimations#43]
I am having an issue changing state with `reverse` prop. 

Headless ui provides an `open` state in Popover component while active and I am providing it to `reverse` prop. 

However, whenever i close the Menu clicking outside of the screen not the UseAnimation provided icon, the Menu closes but the icon stays unchanged.

```jsx
<Popover.Button>
    <div>
        <ul>
            <UseAnimations
                animation={menu}
                size={40}
                speed={2}
                reverse={open}
            />
        </ul>
    </div>
</Popover.Button>
```

**Comment by babur001** [comment]
Can be checked [here](https://www.codify.uz/course)

**Comment by marekfeikus** [comment]
Fixed by #44
